src/ops/cli/parser.py

```diff
@@ -52,14 +54,14 @@ def _check_args_for_unicode(args):
 
         try:
             for value in args:
-                if isinstance(value, unicode):
+                if not PY3 and isinstance(value, unicode):
                     # Python3 or some Python3 compatibility mode can make arguments to be unicode, not str
                     value.encode('utf-8').encode('utf-8')
                 elif isinstance(value, str):  # Python 2 str, check if it can be represented in utf8
                     value.encode('utf-8')
         except UnicodeDecodeError as e:
-            print 'Invalid character in argument "{0}", most likely an "en dash", replace it with normal dash -'.format(
```

Comment:
[not important] Maybe we can add an automatic replacement .replace(u'\u2014', '-', inplace=True) to skip this exception? Check if argument contains en dash and replace with with normal dash. This will not make the user retype the command.